scripts/release/msigdb-gsea/build_gsea.pl

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+our $VERSION = 1.0;
+
+use autodie qw/:all/;
+use Capture::Tiny ':all';
+use Cwd;
+
+my %repositories = (
+    'Pathway-Exchange' => {
+        url => 'https://github.com/reactome/Pathway-Exchange.git',
+        ant_xml_file => 'GSEAdeployAsApplication.xml',
+        output => 'GSEAExport.jar'
+    },
+    'CuratorTool' => {
+        url => 'https://github.com/reactome/CuratorTool.git',
+        ant_xml_file => 'WebELVDiagram.xml',
+        output => 'WebELVTool/*.jar'
+    }
+);
+
+# Build required output from each repository
+foreach my $repository_name (keys %repositories) {
+    my $current_dir = getcwd;
+    # Pull latest changes if local repository exists
+    if (-d "$repository_name/.git") {
+        chdir $repository_name;
+        my $stderr = capture_stderr {
+            system 'git pull';
```

Comment:
I would recommend introducing a variable `$REPO_VERSION` and doing `system 'git checkout $REPO_VERSION;` to make sure you are consistently checking out the correct version. The variable could default to "master" if necessary, but I think this approach should be done here, and also after line 42 (`git clone $repository_url"`) to ensure that the code is always on the correct version.

Reply:
Sure, I'll add it in.  For now, it will have to default to master until we introduce tags since the build needs to get the latest code to ensure fixes and updates are used.